### PR TITLE
[upnpcontrol] Prepare dynamic channels and status messages for translation

### PR DESCRIPTION
**Pull request by mherwege**
Signed-off-by: Mark Herwege <mark.herwege@telenet.be>

This PR externalises all dynamic channel names and labels and all thing status messages, so they can be translated.

### CODE REVIEW

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java [hunk 1]
```diff
@@ -165,8 +165,8 @@ public void dispose() {
     protected void initJob() {
         synchronized (jobLock) {
             if (!upnpIOService.isRegistered(this)) {
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                        "UPnP device with UDN " + getUDN() + " not yet registered");
+                String msg = String.format("@text/offline.device-not-registed [ \"%s\" ]", getUDN());
```
Comment (jlaur) [inline bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java:168]: Doesn't match actual key:
```suggestion
                String msg = String.format("@text/offline.device-not-registered [ \"%s\" ]", getUDN());
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Spotted only one key which seems to mismatch.

File: bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java (hunk 1 above)
Comment (mherwege) [reply bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java]: Many thanks and fixed. There was a second place with the mismatch for the same key.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**